app/assets/javascripts/discourse/models/composer.js.es6

```diff
@@ -195,6 +195,11 @@ const Composer = RestModel.extend({
     return canEditTopicFeaturedLink ? !this.siteSettings.topic_featured_link_onebox : true;
   },
 
+  @computed("topic")
+  draftTitle(topic) {
```

Comment:
I don't think you should add formatting to the model. The model should be where the data lives, not its presentation.

Couldn't you do all this in the template?

```handlebars
{{#if model.topic}}
  {{fa-icon "reply"}} {{model.topic.title}}
{{else}}
  {{i18n "composer.saved_draft"}}
{{/if}}
```